### ISSUE TITLE
value-list-item tabindex issue

### ISSUE THREAD

**Issue by itsminj** [Esri/calcite-components#5671]
### Summary

When multiple value-list-items are created, the first one has a tabindex of 0, but all other value-list-items default to -1

Codepen: https://codepen.io/mjkim12/pen/BaxgEbq

### Actual Behavior

Users are not able to tab from value-list-item to another value-list-item

### Expected Behavior

Users should be able to tab into other value-list-items

### Reproduction Sample

https://codepen.io/mjkim12/pen/BaxgEbq

### Reproduction Steps

1. Go onto the Codepen
2. Tab into a value-list-item
3. Try to tab into the next value-list-item (not possible)

### Reproduction Version

beta.97

### Relevant Info

Chrome

### Esri team

ArcGIS Online

**Comment by jcfranco** [comment]
This is by design and is [described in the component's documentation](https://developers.arcgis.com/calcite-design-system/components/value-list/#keyboard-navigation). Value/pick list items can be navigated via the up/down keys. Tab allows users to move through items within the active list item.
